mlflow/tracking/fluent.py

```diff
@@ -988,6 +989,100 @@ def log_image(image: Union["numpy.ndarray", "PIL.Image.Image"], artifact_file: s
     MlflowClient().log_image(run_id, image, artifact_file)
 
 
+def log_table(
+    data: Union[Dict[str, Any], pd.DataFrame],
+    local_path: Optional[str] = None,
+) -> None:
+    """
+    Log a table to MLflow artifact store.
+
+    :param data: Dictionary or pandas.DataFrame to log.
+    :param local_path: Path to the file to write.
+    :return: None
+
+    .. test-code-block:: python
+        :caption: Dictionary Example
+
+        import mlflow
+
+        table_dict = {
+            "inputs": ["What is MLflow?", "What is Databricks?"],
+            "outputs": ["MLflow is ...", "Databricks is ..."],
+            "toxicity": [0.0, 0.0],
+        }
+
+        with mlflow.start_run():
+            # Log the dictionary as a table
+            mlflow.log_table(data=table_dict, local_path="qabot_eval_results.csv")
+
+    .. test-code-block:: python
+        :caption: Pandas DF Example
+
+        import mlflow
+        import pandas as pd
+
+        table_dict = {
+            "inputs": ["What is MLflow?", "What is Databricks?"],
+            "outputs": ["MLflow is ...", "Databricks is ..."],
+            "toxicity": [0.0, 0.0],
+        }
+        df: pd.DataFrame = pd.DataFrame.from_dict(table_dict)
+
+        with mlflow.start_run():
+            # Log the df as a table
+            mlflow.log_table(data=df, local_path="qabot_eval_results.csv")
+    """
+    if isinstance(data, Dict):
```

Comment:
```suggestion
    if isinstance(data, dict):
```

- `isinstance(data, Dict)`: makes me wonder "Does this really work...?"
- `isintance(data, dict)`: 100% sure this works.